package.json

```diff
@@ -17,8 +17,8 @@
     "typecheck": "turbo run typecheck",
     "knip": "knip",
     "validate-config": "turbo run validate-config",
-    "merge-checks": "turbo run publint manypkg typesync knip typecheck lint validate-config check-deduped-deps are-the-types-wrong",
```

Comment:
why?

Reply:
It can't be run in parallel, as it's not done in memory.